### DISCUSSION TITLE
2.0.4 version

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#16] [Announcements]
- Added change category icon
- Added click tray icon to open window
- Fixed some bugs

<hr /><em>This discussion was created from the release <a href='https://github.com/tonylu110/uyou-todo-electron/releases/tag/2.0.4'>2.0.4 version</a>.</em>

**Comment by JustinLee-2020** [comment]
创建任务时，选择下午提醒时间后，仍然按12小时制显示，且没有pm字样。

**Reply by tonylu110** [reply]
这个问题后续会解决了，应该在下个版本

**Comment by liufang185** [comment]
WIN7有问题。报错2.0.3和2.0.4 都不行啊 ，，报错：“无法定位程序输入点DiscardVirtualMemory于动态链接库KERNEL32.dll，求解决

**Reply by tonylu110** [reply]
win7 已无法使用
